File: frontend/src/dto/TimeDTO.ts
export class TimeDTO{
    private nameInstruction: string;
    private item: string;
    private effectiveDuration: number;
    private expectedDuration: number;
    private result: object;

    constructor(nameInstruction: string, item: string, effectiveDuration: number, expectedDuration: number, result: object){
        this.nameInstruction = nameInstruction;
        this.item = item;
        this.effectiveDuration = effectiveDuration;
        this.expectedDuration = expectedDuration;
        this.result = result;
    }

}

File: backend/src/main/java/com/mima/dto/TimeDTO.java
package com.mima.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString

public class TimeDTO {
	private String nameInstruction;
	private String item;
	private int effectiveDuration;
	private int expectedDuration;
	private String result;

}
